TWLight/resources/views.py

```diff
@@ -543,3 +543,71 @@ def get_redirect_url(self, *args, **kwargs):
             else:
                 obj.upvoted_users.add(user)
         return url_
+
+
+@method_decorator(login_required, name="post")
+class SuggestionMergeView(CoordinatorsOnly, FormView):
+
+    model = Suggestion
+    template_name = "resources/merge.html"
+    form_class = SuggestionMergeForm
+    success_url = reverse_lazy("suggest")
+
+    def get_total_upvotes(self, suggestions):
+
```

Comment:
Please add a docstring to this. You can check an example of a docstring [here](https://github.com/WikipediaLibrary/TWLight/blob/master/TWLight/users/helpers/editor_data.py#L43)